packages/dds/migration-shim/src/migrationShim.ts

```diff
@@ -68,6 +71,12 @@ export class MigrationShim extends TypedEventEmitter<IMigrationEvent> implements
 		super();
 	}
 
+	private _oldTree: LegacySharedTree | undefined;
+	private get oldTree(): LegacySharedTree {
```

Comment:
Let's use "legacyTree" rather than "oldTree" I think.  Or "original" vs. "migrated" if you prefer - same comment for factory naming